ProjectBank/Server/appsettings.Development.json

```diff
@@ -1,5 +1,4 @@
 ﻿{
-  "DetailedErrors": true,
```

Comment:
I guess this line could be relevant to add during development according to https://docs.microsoft.com/en-us/aspnet/core/blazor/fundamentals/handle-errors?view=aspnetcore-6.0